### DISCUSSION TITLE
Improving options for accessing & positioning RQ Devtools

### DISCUSSION THREAD

**Discussion by DevDW** [TanStack/query#3346] [Ideas]
Hello 👋 

For the React Query Devtools, there are currently two modes for their placement in an application: [floating and embedded](https://react-query.tanstack.com/devtools).

In general these options are sufficient, but depending on the particularities of your application and your own preferences, both options can feel awkward or cumbersome at times.

One possibility is a browser extension, which would allow you to use the RQ Devtools within the browser devtools, although there are [no plans for that presently](https://github.com/tannerlinsley/react-query-devtools/issues/23).

A happy medium between the current options and the possibly-far-off browser extension might be an option to pop out the RQ Devtools. How feasible might this be?

**Comment by TkDodo** [comment]
pop out where? To a different browser tab?
